tee/tf/main.tf

```diff
@@ -0,0 +1,277 @@
+terraform {
+  backend "s3" {
+    bucket = "proving-server-tf-state"
+    key    = "terraform/tee/terraform.tfstate"
+    region = "eu-north-1"
+  }
+}
+
+variable "aws_region" {
+  description = "AWS region to deploy to"
+  type        = string
+  default     = "eu-north-1"
+}
+
+variable "instance_type" {
+  description = "EC2 instance type for Nitro Enclave"
+  type        = string
+  default     = "c5.xlarge"
+}
+
+variable "key_name" {
+  description = "(Optional) EC2 key pair name for SSH access"
+  type        = string
+  default     = ""
+}
+
+variable "eif_url" {
+  description = "Public URL of the Enclave Image File (EIF)"
+  type        = string
+}
+
+variable "proxy_image" {
+  description = "Public ECR URI of the proxy Docker image"
+  type        = string
+}
+
+variable "debug_mode" {
```

Comment:
When using debug mode, currently nothing is logged in the enclave logs from the process itself, since this line is https://github.com/Cardinal-Cryptography/blanksquare-monorepo/blob/prover-server-v0.1.0/tee/nix/enclave.nix#L28 empty. We should set there at least `RUST_LOG=debug` I think, so I hope 

```
env = "RUST_LOG=debug";
```

would work

Reply:
I'\ll create a task for it